### PR TITLE
Add Ancestry

### PR DESCRIPTION
**Pull request by bdonvr**

### CODE REVIEW

**Review by Carlgo11 — DISMISSED** [review]

**Review by kmpoppe — CHANGES_REQUESTED** [review]
Please add an exception text. 2FA is currently only available for customers with an US based mobile phone number. Other country sites can't enable it.
//Kai

File: _data/other.yml [hunk 1]
```diff
@@ -28,6 +28,13 @@ websites:
       sms: Yes
       doc: https://support.airtable.com/hc/en-us/articles/219409917-Setting-up-two-factor-authentication
 
+    - name: Ancestry
+      url: https://www.ancestry.com
+      img: ancestry.png
+      tfa: Yes
+      sms: Yes
+      doc: https://support.ancestry.com/s/article/MFA
+
```
Comment (jtagcat) [inline _data/other.yml:39]:
```suggestion
      exceptions:
          text: "2FA is currently only available for customers with an US based mobile phone number."

```

**Review by jtagcat — CHANGES_REQUESTED** [review]

**Review by kmpoppe — DISMISSED** [review]

File: _data/other.yml [hunk 2]
```diff
@@ -28,6 +28,15 @@ websites:
...
       doc: https://support.airtable.com/hc/en-us/articles/219409917-Setting-up-two-factor-authentication
 
+    - name: Ancestry
+      url: https://www.ancestry.com
+      img: ancestry.png
+      tfa: Yes
+      sms: Yes
+      doc: https://support.ancestry.com/s/article/MFA
+      exceptions:
+          text: "2FA is currently only available for customers with an US based mobile phone number."
```
Comment (Carlgo11) [inline _data/other.yml]:
```suggestion
          text: "2FA is currently only available for customers with a U.S. based phone number."
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

File: _data/other.yml (hunk 2 above)
Comment (kmpoppe) [reply _data/other.yml]: @bdonvr As you have not allowed other users to commit to your branch, would you please add this code suggestion? 
Thanks! 
// Kai

**Review by kmpoppe — APPROVED** [review]

**Review by Carlgo11 — APPROVED** [review]

**Pull request merged**